samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,393 @@
+
+To use this configuration, include the ``-DCONFIG_APP_VERSION=x`` option with the build command, where x is the version number.
+
+For example:
+
+.. code-block:: console
+
+   west build -p -b nrf54h20dk/nrf54h20/cpuapp -- -DCONFIG_APP_VERSION=2
+
+If you do not specify this configuration, the sample is built with Version 1.
+
+Configuration options
+=====================
+
+Check and configure the following configuration option for the sample:
+
+.. _CONFIG_APP_VERSION:
+
+CONFIG_APP_VERSION - Configuration for the application version
+   The sample configuration updates the version of the application.
+   The default value is ``1``.
+
+Modify partition sizes
+======================
+
+You can also modify the size and location of the partitions.
+This is done by modifying the values for the desired location and size of the partition in the devicetree :file:`.overlay` files.
+
+* To modify the Application Core’s partition size,  modify the values for ``slot0_partition`` defined in :file:`modules/ic-next/dts/arm/nordic_nrf_next/nrf54h20_cpuapp.dtsi`.
+
+* To modify the DFU partition, modify the values for ``dfu_partition`` defined in :file:`samples/suit/smp_transfer/boards/nrf54h20dk_nrf54h20_cpuapp.overlay`.
+
+    * ``dfu_partition`` is where the update candidate is stored before the update process begins.
+
+Manifest template files
+=======================
+
+All required manifest template files are automatically created after the sample is built for the first time.
+These are needed to create a SUIT envelope to carry out the DFU.
+
+.. note::
+
+   The Radio Domain manifest template is only created when building the Bluetooth Low Energy version of the sample, and not the UART version.
+   Currently, it is not needed for the UART version.
+
+If you want to make modifications to how the DFU is executed in this sample, you can do so by editing the manifest templates.
+See the :ref:`ug_nrf54h20_suit_customize_dfu` user guide for instructions and examples.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/suit/smp_transfer`
+
+This sample can be found under |sample path| in the |NCS| folder structure.
+
+.. caution::
+   In specific instances, after a cold boot, the nRF54H20 DK can behave like a bricked device (being unreachable via debugger, or non-booting).
+   To fix the issue, briefly tap (for less than 0.5 seconds) the Reset button on the DK.
+
+Building and programming using the command line
+===============================================
+
+To build and program the sample to the nRF54H20 DK, complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: Over Bluetooth Low Energy
+
+      1. Open a terminal window in |sample path|.
+      #. Build the sample using the following ``west`` command, with the following Kconfig options set:
+
+         .. code-block:: console
+
+            west build -p -b nrf54h20dk/nrf54h20/cpuapp -- -DOVERLAY_CONFIG=overlay-bt.conf -DCONFIG_APP_VERSION=1
```

Comment:
In NCS we use:
 - `CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM` instead of `CONFIG_APP_VERSION`
 - `-DFILE_SUFFIX=bt` instead of `-DOVERLAY_CONFIG=overlay-bt.conf`
 - All of the `west build` commands must provide `--sysbuild` argument before `--` delimiter